src/graph_notebook/magics/metadata.py

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
Can we get some tests in place for this module? Maybe add in a helper function that builds the metadata object from a sparql query, and another from a gremlin query? We want to ensure that we can build the right object and testing the magic commands are a little tough. They act more like a blackbox.

Reply:
Done, added functions to initialize the metadata objects and build them out of Gremlin/Sparql queries. Integration tests added for these functions.